README.md

```diff
@@ -69,6 +70,33 @@ jobs:
     - name: Cache workaround # https://github.com/actions/cache/issues/133
       run: chmod -R a+rwx .
 ```
+You can also use the provided Github Action:
```

Comment:
Maybe we should add a comment saying caching may probably not work when using docker-android as action?